patches/react-native-collapsible+1.6.0.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/react-native-collapsible/index.d.ts b/node_modules/react-native-collapsible/index.d.ts
```

Comment:
Is there an upstream issue tracking this (or a PR open to their repo to fix it)?

Reply:
there are both :)